src/nitbuilder.nit

```diff
@@ -0,0 +1,286 @@
+# This file is part of NIT ( http://www.nitlanguage.org ).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Instantiation and transformation of language entities in AST and model.
```

Comment:
```suggestion
# Create AST nodes from Model entities
#
# TODO explain briefly why this is useful
# TODO give a short `nitish` example about how I can create a AMethPropDef from a MPropdef 
```

Reply:
The module does not only allow to build AST nodes from the model. It also makes it possible to completely construct elements of the language (representation in the model and the ast).